### DISCUSSION TITLE
Preserving Original File Modification Times during WebDAV Sync

### DISCUSSION THREAD

**Discussion by rcsoul** [remotely-save/remotely-save#525] [General]
Hi,

I'm encountering an issue while syncing files to WebDAV. The "Modified time" attributes of all files are getting changed to the time of the initial sync. This behavior effectively replaces the original modification timestamps with the sync timestamp.

I'm reaching out to inquire if there's a method to maintain the original "Modified time" attributes during the synchronization process. Preserving these timestamps is crucial for maintaining accurate file histories and managing version control.

If there's a specific setting or configuration option within WebDAV or Remotely Save to achieve this, I would appreciate any guidance on enabling it.

Thank you for your time and assistance.

**Comment by fyears** [comment]
sorry but that's almost not possible because webdav spec only keeps upload time.
